### PR TITLE
 KeepKey: Fix WebUSB on Windows 10 1903

### PR DESCRIPTION
**Pull request by EchterAgo**
* Updates libusb to a patched version that can ignore `LIBUSB_ERROR_ACCESS` when opening the HID part of a composite device fails.
* Changes the KeepKey plugin to use libusb for enumeration directly instead of using transport_webusb.

### CODE REVIEW

File: plugins/keepkey/keepkey.py [hunk 1]
```diff
@@ -86,40 +87,45 @@ def __init__(self, parent, config, name):
             import keepkeylib.ckd_public
             import keepkeylib.transport_hid
             import keepkeylib.transport_webusb
+            from electroncash_plugins.hw_wallet.utils.libusb import USBContext
```
Comment (cculianu) [inline plugins/keepkey/keepkey.py]: Hey the rest of this file uses relative imports (eg `..hw_wallet`).  I am not clear on the caveats of each -- why did you opt to do it this way?

Comment (EchterAgo) [reply plugins/keepkey/keepkey.py]: No reason at all, should I change it to relative?

Comment (cculianu) [reply plugins/keepkey/keepkey.py]: Yeah no idea what the caveats are.. figured keep it same same.  Thanks. 👍

Comment (EchterAgo) [reply plugins/keepkey/keepkey.py]: Changed to relative.

**Comment by EchterAgo** [comment]
I removed the custom USB context and enabled the libusb patch by default because that also fixes Trezor on Windows.

**Comment by cculianu** [comment]
Nice work man!  Works great for both Trezor and KeepKey on Win10 1903, Win10 1803, etc.

Didn't break for Linux or MacOS  Good work.

**Comment by EchterAgo** [comment]
Also tested on Windows 7. You need to use [Zadig](https://zadig.akeo.ie/) to install the WinUSB driver but then it works.

**Pull request merged**